internal/config/storageclass/storage-class.go

```diff
@@ -40,14 +43,23 @@ const (
 const (
 	ClassStandard = "standard"
 	ClassRRS      = "rrs"
-	ClassOptimize = "optimize"
+	Optimize      = "optimize"
+	InlineBlock   = "inline_block"
 
 	// Reduced redundancy storage class environment variable
 	RRSEnv = "MINIO_STORAGE_CLASS_RRS"
 	// Standard storage class environment variable
 	StandardEnv = "MINIO_STORAGE_CLASS_STANDARD"
 	// Optimize storage class environment variable
 	OptimizeEnv = "MINIO_STORAGE_CLASS_OPTIMIZE"
+	// Inline block indicates the size of the shard
+	// that is considered for inlining, remember this
+	// shard value is the value per drive shard. It
+	// will vary based on the parity that is configured
+	// for the STANDARD storage_class.
+	// inlining means data and metadata are written
+	// together in a single file i.e xl.meta
+	InlineBlockEnv = "MINIO_STORAGE_CLASS_INLINE_BLOCK"
```

Comment:
```suggestion
	InlineBlockEnv = "_MINIO_STORAGE_CLASS_INLINE_BLOCK"
```

We should not make this an official env var. It should not be needed. We are not C*ph - we should figure out where the difference lies instead of just pushing the problem to the user.